scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml

```diff
@@ -199,6 +199,41 @@ HTSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.4x7.8mm_Mask2.4x4.68mm_ThermalVias:
     # bottom_pad_size:
     paste_avoid_via: False
 
+HTSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP2.85x5.4mm:
```

Comment:
You do not need to duplicate the non thermal vias version. (if thermal vias section is given then non thermal via version is generated as well)

Reply:
Done